### PR TITLE
Text-related changes from feedback during unofficial reading

### PR DESCRIPTION
**Pull request by rzambre**

### CODE REVIEW

**Comment by garzaran** [comment]
This looks fine to me.

**Review by garzaran — APPROVED** [review]

File: mem_alloc.tex [hunk 1]
```diff
@@ -52,27 +52,28 @@ \chapter{Overview}
...
 \chapter{Definitions}
 \label{chap:definitions}
 
-This section contains definitions of memory allocation kinds and
-their restrictors.
+This chapter contains definitions of memory allocation kinds and
+their restrictors for different memory types.
 
-\section{Kind: cuda}
+\section{CUDA memory kinds}
```
Comment (Wee-Free-Scot) [inline mem_alloc.tex:73]:
```suggestion
\section{CUDA memory kind}
```
There is one kind (although there are several restrictors for that kind).

File: mem_alloc.tex [hunk 2]
```diff
@@ -96,10 +97,10 @@ \subsubsection{Restrictors}
 
 \end{itemize}
 
-\section{Kind: rocm}
+\section{ROCm memory kinds}
```
Comment (Wee-Free-Scot) [inline mem_alloc.tex:100]:
```suggestion
\section{ROCm memory kind}
```
There is one kind (although there are several restrictors for that kind).

File: mem_alloc.tex [hunk 3]
```diff
@@ -120,10 +121,10 @@ \subsubsection{Restrictors}
 
 \end{itemize}
 
-\section{Kind: level\_zero}
+\section{Level Zero memory kinds}
```
Comment (Wee-Free-Scot) [inline mem_alloc.tex:124]:
```suggestion
\section{Level Zero memory kind}
```
There is one kind (although there are several restrictors for that kind).

**Review by Wee-Free-Scot — CHANGES_REQUESTED** [review]
kinds -> kind for each kind
Otherwise LGTM.

**Review by edgargabriel — APPROVED** [review]
Looks fundamentally good to me. 

Short question though: the update to the ROCm example to use MPI_Comm_get_info on MPI_COMM_WORLD instead, of MPI_INFO_ENV: shall I wait until we have an agreement on the errata, or should I go ahead and file it?

**Comment by rzambre** [comment]
> Short question though: the update to the ROCm example to use MPI_Comm_get_info on MPI_COMM_WORLD instead, of MPI_INFO_ENV: shall I wait until we have an agreement on the errata, or should I go ahead and file it?

I think we should modify the File example to use MPI_Comm_get_info on MPI_COMM_WORLD. The change would remove the dependency on the status of the MPI_INFO_ENV errata.

**Comment by garzaran** [comment]
> > Short question though: the update to the ROCm example to use MPI_Comm_get_info on MPI_COMM_WORLD instead, of MPI_INFO_ENV: shall I wait until we have an agreement on the errata, or should I go ahead and file it?
> 
> I think we should modify the File example to use MPI_Comm_get_info on MPI_COMM_WORLD. The change would remove the dependency on the status of the MPI_INFO_ENV errata.

that might be a good idea ..

**Comment by edgargabriel** [comment]
done, please see https://github.com/mpi-forum/mem-alloc/pull/12

**Pull request merged**